### DISCUSSION TITLE
Booting problem

### DISCUSSION THREAD

**Discussion by hatamiarash7** [plausible/analytics#3558] [Self-Hosted Support]
### Past Issues Searched

- [X] I have searched open and closed issues to make sure that the bug has not yet been reported

### Issue is a Bug Report

- [X] This is a bug report and not a feature request, nor asking for self-hosted support

### Using official Plausible Cloud hosting or self-hosting?

Self-hosting

### Describe the bug

There is an error when starting Plausible:

```
ERROR! Config provider Config.Reader failed with:
** (ArgumentError) errors were found at the given arguments:

  * 1st argument: not an already existing atom

    :erlang.binary_to_existing_atom("True", :utf8)
    /app/releases/0.0.1/runtime.exs:189: (file)
    (elixir 1.14.3) src/elixir.erl:309: anonymous fn/4 in :elixir.eval_external_handler/1
    (stdlib 4.2) erl_eval.erl:748: :erl_eval.do_apply/7
    (stdlib 4.2) erl_eval.erl:492: :erl_eval.expr/6
    (stdlib 4.2) erl_eval.erl:136: :erl_eval.exprs/6
    (elixir 1.14.3) src/elixir.erl:294: :elixir.eval_forms/4
    (elixir 1.14.3) lib/module/parallel_checker.ex:110: Module.ParallelChecker.verify/1
```

### Expected behavior

-

### Environment

```markdown
- OS: Ubuntu - Docker
```

**Comment by ruslandoga** [comment]
👋 @hatamiarash7 

Seems like you set some `plausible-conf.env` env var to `True` (instead of `true`) and it's not allowed :)
